modules/promotions/assets/js/react/components/promotion-card.js

```diff
@@ -36,10 +38,16 @@ const PromotionCard = ( { doClose, promotionsData } ) => {
 					} } onClick={ doClose } />
 				</Stack>
 				<Image src={ imgSrc } alt={ imgAlt } sx={ { height: 150, width: '100%' } } />
-				<Stack py={ 1 } px={ 2 }>
-					{ description.map( ( e, index ) => {
-						return ( <Typography key={ index } variant="body2" color="secondary">&bull; { e }</Typography> );
-					} ) }
+				<Stack px={ 2 }>
+					<List sx={ { listStyle: 'disc', listStylePosition: 'outside', pl: 1 } }>
+						{ description.map( ( e, index ) => {
+							return (
+								<ListItem key={ index } sx={ { listStyle: 'disc', display: 'list-item', color: 'text.secondary', p: 0 } }>
```

Comment:
a small question on my side, we had to duplicate the `listStyle: 'disc'` for the list and for the item (were we forced to do that? or will it work with just one of them?)